randovania/games/am2r/json_data/Industrial Complex.txt

```diff
@@ -814,7 +824,7 @@ Extra - map_name: rm_a3a14
   * Layers: default
   * Morph Ball Tunnel to Upper Factory Alpha Nest/Tunnel to Upper Factory Alpha Nest Access
   > Dock to Spiky Shaft
-      Power Grip Wall
+      Can Use Any Bombs and Power Grip Climb
```

Comment:
bombs are not needed, you spawn outside of the block
Also this should be kept as power grip wall